clap-v3-utils/src/input_parsers/signer.rs

```diff
@@ -110,6 +113,17 @@ impl SignerSource {
                 source.to_string()
             }
         };
+
+        // If the user supplies a base-58 encoded keypair accept it as a valid SignerSource
+        if bs58::decode(&source)
+            .into_vec()
+            .ok()
+            .and_then(|bytes| Keypair::from_bytes(&bytes).ok())
+            .is_some()
```

Comment:
nit: Are the `ok()`s needed? ie can you do
```suggestion
        if bs58::decode(&source)
            .into_vec()
            .and_then(|bytes| Keypair::from_bytes(&bytes))
            .is_ok()
```